Slider/Assets/Scripts/UI/Settings/IntSettingUIElement.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class IntSettingUIElement : SettingUIElement<int>
+{
+
+}
```

Comment:
Unity serialization sucks x3

Reply:
this should probably be mentioned in a comment x3